Examples/Resources/Extractors/SAT-features-competition2024/sparkle_extractor_wrapper.py

```diff
@@ -0,0 +1,211 @@
+    "unit-featuretime": (FeatureGroup.UNIT, FeatureType.FEATURE_TIME),
+    "saps_BestSolution_Mean": (FeatureGroup.LS, "saps_BestSolution_Mean"),
+    "saps_BestSolution_CoeffVariance": (FeatureGroup.LS, "saps_BestSolution_CoeffVariance"),
+    "saps_FirstLocalMinStep_Mean": (FeatureGroup.LS, "saps_FirstLocalMinStep_Mean"),
+    "saps_FirstLocalMinStep_CoeffVariance": (FeatureGroup.LS, "saps_FirstLocalMinStep_CoeffVariance"),
+    "saps_FirstLocalMinStep_Median": (FeatureGroup.LS, "saps_FirstLocalMinStep_Median"),
+    "saps_FirstLocalMinStep_Q.10": (FeatureGroup.LS, "saps_FirstLocalMinStep_Q.10"),
+    "saps_FirstLocalMinStep_Q.90": (FeatureGroup.LS, "saps_FirstLocalMinStep_Q.90"),
+    "saps_BestAvgImprovement_Mean": (FeatureGroup.LS, "saps_BestAvgImprovement_Mean"),
+    "saps_BestAvgImprovement_CoeffVariance": (FeatureGroup.LS, "saps_BestAvgImprovement_CoeffVariance"),
+    "saps_FirstLocalMinRatio_Mean": (FeatureGroup.LS, "saps_FirstLocalMinRatio_Mean"),
+    "saps_FirstLocalMinRatio_CoeffVariance": (FeatureGroup.LS, "saps_FirstLocalMinRatio_CoeffVariance"),
+    "ls-saps-featuretime": (FeatureGroup.LS, "ls-saps-featuretime"),
+    "gsat_BestSolution_Mean": (FeatureGroup.LS, "gsat_BestSolution_Mean"),
+    "gsat_BestSolution_CoeffVariance": (FeatureGroup.LS, "gsat_BestSolution_CoeffVariance"),
+    "gsat_FirstLocalMinStep_Mean": (FeatureGroup.LS, "gsat_FirstLocalMinStep_Mean"),
+    "gsat_FirstLocalMinStep_CoeffVariance": (FeatureGroup.LS, "gsat_FirstLocalMinStep_CoeffVariance"),
+    "gsat_FirstLocalMinStep_Median": (FeatureGroup.LS, "gsat_FirstLocalMinStep_Median"),
+    "gsat_FirstLocalMinStep_Q.10": (FeatureGroup.LS, "gsat_FirstLocalMinStep_Q.10"),
+    "gsat_FirstLocalMinStep_Q.90": (FeatureGroup.LS, "gsat_FirstLocalMinStep_Q.90"),
+    "gsat_BestAvgImprovement_Mean": (FeatureGroup.LS, "gsat_BestAvgImprovement_Mean"),
+    "gsat_BestAvgImprovement_CoeffVariance": (FeatureGroup.LS, "gsat_BestAvgImprovement_CoeffVariance"),
+    "gsat_FirstLocalMinRatio_Mean": (FeatureGroup.LS, "gsat_FirstLocalMinRatio_Mean"),
+    "gsat_FirstLocalMinRatio_CoeffVariance": (FeatureGroup.LS, "gsat_FirstLocalMinRatio_CoeffVariance"),
+    "ls-gsat-featuretime": (FeatureGroup.LS, "ls-gsat-featuretime"),
+    "lobjois-mean-depth-over-vars": (FeatureGroup.LOBJOIS, "lobjois-mean-depth-over-vars"),
+    "lobjois-log-num-nodes-over-vars": (FeatureGroup.LOBJOIS, "lobjois-log-num-nodes-over-vars"),
+    "lobjois-featuretime": (FeatureGroup.LOBJOIS, FeatureType.FEATURE_TIME),
+    "LP_OBJ": (FeatureGroup.LP, "LP_OBJ"),
+    "LPSLack-mean": (FeatureGroup.LP, "lpslack-mean"),
+    "LPSLack-coeff-variation": (FeatureGroup.LP, "lpslack-coeff-variation"),
+    "LPSLack-min": (FeatureGroup.LP, "lpslack-min"),
+    "LPSLack-max": (FeatureGroup.LP, "lpslack-max"),
+    "lpIntRatio": (FeatureGroup.LP, "lpIntRatio"),
+    "lpTIME": (FeatureGroup.LP, FeatureType.FEATURE_TIME)
+}
+
+
+if args.features:
```

Comment:
I don't like code that is just "floating" in the file, add if __name__ == '__main__: